### ISSUE TITLE
Error: Connection activation failed: (7) Secrets were required, but not provided

### ISSUE THREAD

**Issue by rlelchev** [lwfinger/rtl8188eu#375]
I tried to install this driver and I got it successfully installed, but NetworkManager gives me this error message. Any help?

**Comment by lwfinger** [comment]
Are you sure you gave the secret correctly?

Which branch are you using?  Branch v4.1.8_9499 is currently being added to the kernel, and it works correctly.

**Comment by rlelchev** [comment]
I don't know the fix for this issue, because when I reinstalled my OS everything got fixed. It might have been just me being too much of a noob on Linux to type out the secrets. I was a Linux beginner back then and I didn't know what I was doing. Arch Linux and other minimal Linux distros do not come preinstalled with NetworkManager and everything it requires and that makes me need to install drivers and other things to make everything work. Thank you for trying to help, even though I was such a noob back then.
